releases/20.0.0.11/full/helpers/build/populate_scc.sh

```diff
@@ -9,6 +9,15 @@ SCC_SIZE="80m"  # Default size of the SCC layer.
 ITERATIONS=2    # Number of iterations to run to populate it.
 TRIM_SCC=yes    # Trim the SCC to eliminate any wasted space.
 
+# If this directory exists and has at least ug=rwx permissions, assume the base image includes an SCC called 'openj9_system_scc' and build on it.
+# If not, build on our own SCC.
+if [[ -d "/opt/java/.scc" ]] && [[ `stat -L -c "%a" "/opt/java/.scc" | cut -c 1,2` == "77" ]]
```

Comment:
FYI: there is a discussion whether to embed an SCC into the IBM sdk as well. The location of the IBM SDK is different: `/opt/ibm/java`, so this line might need to be changed if we end up with that SCC in the ibmjava container.